packages/edge-bundler/node/bridge.ts

```diff
@@ -12,9 +12,12 @@ import { getLogger, Logger } from './logger.js'
 import { getBinaryExtension } from './platform.js'
 
 const DENO_VERSION_FILE = 'version.txt'
+
 // When updating DENO_VERSION_RANGE, ensure that the deno version installed in the
 // build-image/buildbot does satisfy this range!
-const DENO_VERSION_RANGE = '^1.37.0'
+// We're pinning the range because of an issue with v1.45.0 of the Deno CLI:
+// https://linear.app/netlify/issue/FRP-775/deno-cli-v1450-causing-issues
+const DENO_VERSION_RANGE = '<1.45.0'
```

Comment:
👀 I believe this syntax would allow 0.x

Reply:
Good point! Updated in https://github.com/netlify/build/pull/5762/commits/92d53f0ac16fcfc783be832a62045a1b56462b46.